ddtrace/tracer/option.go

```diff
@@ -29,6 +30,10 @@ type config struct {
 	// debug, when true, writes details to logs.
 	debug bool
 
+	// logStartup, when true, causes various startup info to be written
+	// when the tracer starts.
+	logStartup bool
```

Comment:
IMHO it's terrible to have this _and_ `debug`. Can't we just have `debug`? This is clearly debug mode... /cc @labbati 

Support will almost always ask for both. Can't we just have a `DD_TRACE_DEBUG` or something?

Reply:
This should be on by default with option to disable and debug logging should *not* be on by default.